ODIN_II/SRC/netlist_utils.cpp

```diff
@@ -443,14 +449,20 @@ void combine_nets(nnet_t* output_net, nnet_t* input_net, netlist_t* netlist) {
  *-------------------------------------------------------------------------------------------*/
 void join_nets(nnet_t* join_to_net, nnet_t* other_net) {
     if (join_to_net == other_net) {
-        if ((join_to_net->driver_pin) && (join_to_net->driver_pin->node != NULL) && join_to_net->driver_pin->node->related_ast_node != NULL)
-            error_message(NETLIST, join_to_net->driver_pin->node->related_ast_node->line_number, join_to_net->driver_pin->node->related_ast_node->file_number, "%s", "This is a combinational loop\n");
-        else
-            error_message(NETLIST, -1, -1, "%s", "This is a combinational loop\n");
-        if ((join_to_net->fanout_pins[0] != NULL) && (join_to_net->fanout_pins[0]->node != NULL) && join_to_net->fanout_pins[0]->node->related_ast_node != NULL)
-            error_message(NETLIST, join_to_net->fanout_pins[0]->node->related_ast_node->line_number, join_to_net->fanout_pins[0]->node->related_ast_node->file_number, "%s", "This is a combinational loop with more info\n");
-        else
-            error_message(NETLIST, -1, -1, "%s", "Same error - This is a combinational loop\n");
+        oassert(join_to_net->num_driver_pins <= 1);
```

Comment:
I don't believe this is needed, It will error out lower, can you turn all the errors bellow into warnings and finally error so we can trace the pins

Reply:
Will do. Just double check looping over fanouts is actually correct

Reply:
Also should I print the driver pin names?
If so would it be `driver_pin[i]->name` or `driver_pin[i]->node->name`?

Reply:
driver_pin[i]->name i believe